src/main/java/org/radarcns/stream/phone/PhoneUsageAggregationStream.java

```diff
@@ -1,42 +1,42 @@
 package org.radarcns.stream.phone;
 
-import java.util.Collection;
+import java.time.Duration;
 import javax.annotation.Nonnull;
 import org.apache.kafka.streams.kstream.KStream;
-import org.radarcns.config.RadarPropertyHandler;
+import org.radarcns.config.RadarPropertyHandler.Priority;
 import org.radarcns.kafka.AggregateKey;
 import org.radarcns.kafka.ObservationKey;
 import org.radarcns.passive.phone.PhoneUsageEvent;
-import org.radarcns.stream.KStreamWorker;
+import org.radarcns.stream.SensorStreamWorker;
 import org.radarcns.stream.StreamDefinition;
-import org.radarcns.stream.StreamMaster;
 import org.radarcns.stream.aggregator.PhoneUsageAggregate;
 import org.radarcns.util.serde.RadarSerdes;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 /**
  * Created by piotrzakrzewski on 26/07/2017.
  */
-public class PhoneUsageAggregationStream extends KStreamWorker<ObservationKey, PhoneUsageEvent> {
-    private static final Logger logger = LoggerFactory.getLogger(PhoneUsageAggregationStream.class);
-
-    public PhoneUsageAggregationStream(Collection<StreamDefinition> definitions, int numThread,
-            StreamMaster master, RadarPropertyHandler properties) {
-        super(definitions, numThread, master, properties, logger);
+public class PhoneUsageAggregationStream extends
+        SensorStreamWorker<ObservationKey, PhoneUsageEvent> {
+    @Override
+    protected void initialize() {
+        defineStream(
+                "android_phone_usage_event_output",
+                "android_phone_usage_event_aggregated",
+                Duration.ofDays(1));
```

Comment:
These should also be windowed as neither the Rest API nor the dashboard has a way of consuming these.

Reply:
I don't see a way to aggregate them though, or how to show them in the Dashboard at all. If there is a suggestion about this, we can add a new implementation for this. These streams are primarily meant to make the output files in HDFS more manageable.

Reply:
But the output topics from this stream are never used in HDFS connector topics config. So thats why i was wondering where is this used.

Reply:
Good point, that is an error, it should definitely be added to the config.

Reply:
See https://github.com/RADAR-base/RADAR-Docker/pull/157.